app/lib/omni_auth/strategies/paid_connect.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: UTF-8
+module OmniAuth
+  module Strategies
+    # PaidConnect strategy for Omniauth
+    class PaidConnect < OmniAuth::Strategies::OAuth2
+      option :name, :paid_connect
+
+      option :client_options, site: 'https://auth.paidlabs.com'
```

Comment:
yeah, actually it's recommended to use `.fetch` instead: `ENV.fetch('PAID_OAUTH_URI', 'https://auth.paidlabs.com')`